polyply/src/gen_seq.py

```diff
@@ -117,24 +167,25 @@ def _add_edges(graph, edges, idx, jdx):
     ----------
     graph: `:class:networkx.Graph`
     edges:  list[str]
-        str has the format `node_keyA,node_keyB-node_keyC,node_keyD`
+        str has the format `node_keyA-node_keyB,node_keyC-node_keyD`,
+        where edges will be added between nodes A and B, and C and D
     idx: int
     jdx: int
 
     Returns
     --------
     `:class:networkx.Graph`
     """
-    for edge in edges.split(","):
-        node_idx, node_jdx = edge.split("-")
+    for edge in edges.strip().split(","):
+        node_idx, node_jdx = edge.strip().split("-")
```

Comment:
```python
for edge in edges.split(','):
    node_idx, node_jdx = map(str.strip, edge.split('-'))
```
No point in stripping edges if you're also stripping edge. And by that logic, if you strip all elements of edge.split you're done.